test/test.js

```diff
@@ -252,6 +253,38 @@ test('multiple hook.unhook()', function (t) {
   t.end()
 })
 
+test('absolute file paths', function (t) {
```

Comment:
It would be nice with an additional test to verify that `require('/foo/bar')` correctly triggers `hook(['/foo/bar/index.js'], ...)`

Reply:
I had not considered that use case. Added in 96e6b6bdd6c549b52e5634cf1ef0426e33852f68